tests/unit/Exception/MissingAppConfigurationExceptionTest.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+use AG\ElasticApmLaravel\Exception\MissingAppConfigurationException;
+use Codeception\Test\Unit;
+
+class MissingAppConfigurationExceptionTest extends Unit
+{
+    /** @var MissingAppConfigurationException */
+    private $exception;
+
+    protected function _before(): void
+    {
+        $this->exception = new MissingAppConfigurationException();
+    }
+
+    public function testExceptionMessage(): void
+    {
+        self::assertEquals('Use AgentBuilder::withAppConfig() to provide a configuration object.', $this->exception->getMessage());
```

Comment:
I'm surprised linter doesn't complain about line length.

Reply:
[From PSR-2](https://www.php-fig.org/psr/psr-2/):

>There MUST NOT be a hard limit on line length; the soft limit MUST be 120 characters; lines SHOULD be 80 characters or less.

I love the contradiction, no hard limit but soft limit MUST be.